github/ipfs.yml

```
@@ -2715,6 +2715,8 @@ repositories:
     description: "An unobtrusive and user-friendly desktop application for IPFS on
       Windows, Mac and Linux. "
     files:
+      .github/workflows/auto-project.yml:
+        content: files/.github/workflows/add_issues_to_gui_project.yml
```

We want to overwrite on create here because the file already exists in `ipfs-desktop`.

```suggestion
        content: files/.github/workflows/add_issues_to_gui_project.yml
        overwrite_on_create: true
```